Makefile

```diff
@@ -69,4 +69,4 @@ all:
 
 install:
 	install -m 644 pc-boot.img pc-core.img shim.efi.signed grubx64.efi $(DESTDIR)/
-	install -m 644 grub.conf grub.cfg $(DESTDIR)/
+	install -m 644 grub.cfg-recovery grub.cfg-normal grub.conf $(DESTDIR)/
```

Comment:
'grub.conf' is not a native name for the config file on grub's side, this name exists for compatibility with snapd's requirements. If there are now two grub.cfg's, should the grub.conf symlink be dropped?

Reply:
I think we can drop the symlink now - I'm not sure about the consequences though - I would like to do that in a followup so that if things break we don't need to look at too many variables.

Reply:
+1 to deferring.